### ISSUE TITLE
please improve documentation for yora-gateway

### ISSUE THREAD

**Issue by torstenwerner** [FranckPachot/yora#1]
Could you please improve the documentation about the settings related to credentials and hostnames?

1. Which environment variables I have to set for the gateway container in particular for credentials?
2. Which value should be set for --hostname?
3. Which values I have to use in the db link: connect to, identified by, and HOST?

I am getting the following error in oracle at the moment:

ORA-28500: connection from ORACLE to a non-Oracle system returned this message: connection to server at "eu-central-***.aws.ybdb.io" (remote ip), port 5433 failed: FATAL: no pg_hba.conf entry for host "local ip", user "admin", database "yugabyte", SSL off {08001,NativeErr = 101} ORA-02063: preceding 3 lines from ... 

I can access the managed yugabytedb directly from my machine but it fails using the gateway and the db link in oracle.

**Comment by FranckPachot** [comment]
Yes, I should explain them. In short:
1. user and password are passed by the dblink but other settings should be passed as environment variables
2. `--hostname` is used to reference if from other containers (for example when running Oracle XE on a container the db link addresses the hostname). In my example I used the IP (which you can get with `docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' yora-gateway`)
3. `connect to` and `identified by`are the postgres user and password. `HOST`is the connection to the gateway

YugabyteDB Managed requires SSL and your message shows `SSL off`. I think you need to start the gateway with `-e PGSSLMODE=require` but I haven't tested. I hope it works.

**Comment by torstenwerner** [comment]
Thank you for your explanation. The parameter `--hostname yora-gateway` is useful with `--link yora-gateway` for the xe container.

Setting `-e PGSSLMODE=require` for the gateway container does not help yet. I am still getting the same error message including `SSL off`.

**Comment by FranckPachot** [comment]
Yes apparently PGSSLMODE is not read by the ODBC driver. This can be worked around by putting it in `/etc/odbc.ini`
```
[ODBC Data Sources]
YORA =

[YORA]
Driver = PostgreSQL
Sslmode=allow
```
I'll update the code, image and doc. thanks for reporting

**Comment by FranckPachot** [comment]
Should be ok now
